snippets/logging-get-clusterid-snip.adoc

```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following modules and assemblies:
+//
+
+:_content-type: SNIPPET
+
+Logs from any source contain a clusterID, the unique identifier of the cluster in which the Operator is deployed.
```

Comment:
Do you think this sounds better? 
_contains a field 'openshift.cluster_id', the unique identifier of the cluster in which the Operator is deployed._